pkg/controller/nstemplateset/nstemplateset_controller.go

```diff
@@ -0,0 +1,260 @@
+package nstemplateset
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/condition"
+	"github.com/go-logr/logr"
+	"github.com/operator-framework/operator-sdk/pkg/predicate"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	errs "github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
```

Comment:
it seems like we are not consistent in how we organize our imports - perhaps we should come up with an agreement so we aren't all doing this differently? see this PR for instance: https://github.com/codeready-toolchain/member-operator/pull/49/files#diff-ae1e62ddb6744c2036a1b470ddc6a527R4

Reply:
@tinakurian if you wish, you can come up with some proposal and come to some agreement with team.  But all this should be out of this PR.